support-frontend/assets/components/subscriptionCheckouts/endSummary/endSummary.jsx

```diff
@@ -2,38 +2,30 @@
 
 import React from 'react';
 import * as styles from './endSummaryStyles';
-import { type Option } from 'helpers/types/option';
 
 const Dot = () => <div css={styles.dot} />;
 
 function EndSummary(props: {
```

Comment:
Is this component just for Digital Subscriptions? If it is it would probably be better to put it in the components related to that checkout rather than the top level `components\subscriptionsCheckouts` that is supposed to be for components which are common to all the checkouts

Reply:
It is just for digital subscriptions at the moment. I felt it was best to keep it specific for its purpose for the moment (in the spirit of YAGNI) and adapt it to a wider use later when that is required. But yes, the placement of it in this folder is called into question by taking that course. I will move it.